config/application.rb

```diff
@@ -114,13 +114,6 @@ class Application < Rails::Application
     # on the top to bottom order of initializer calls in the file.
     # Because this is easy to mess up, keep your initializers in order.
     # register plugins even before loading settings, as plugins can bring their own settings
-    initializer :register_vmdb_plugins, :before => :load_vmdb_settings do
-      Rails.application.railties.each do |railtie|
-        next unless railtie.class.name.start_with?("ManageIQ::Providers::") || railtie.try(:vmdb_plugin?)
-        Vmdb::Plugins.instance.register_vmdb_plugin(railtie)
```

Comment:
I'm fine with moving the logic to `Vmdb::Plugins#initialize`, but I think we would minimally need to call `Vmdb::Plugins.instance` in this initializer.  Otherwise, I don't think the plugins will have registered by the time `Vmdb::Settings.init` is called, right?